source/class/qx/bom/client/Html.js

```diff
@@ -353,7 +353,7 @@ qx.Bootstrap.define("qx.bom.client.Html",
      * @ignore(Image)
      */
     getDataUrl : function(callback) {
-      var data = new Image();
+      var data = new window.Image();
```

Comment:
With the advent of DOM simulation in server-side JS (Node), maybe it would make sense to avoid use of `window` and use, instead, `globalThis` which is implemented in all of the modern JS engines, and presumably mapped as necessary when transpiled, as well. I have found `globalThis` to be very useful when running the same code both client- and server-side.